utils/_context/_scenarios/__init__.py

```diff
@@ -651,6 +651,14 @@ def all_endtoend_scenarios(test_object):
         github_workflow="libinjection",
     )
 
+    container_auto_injection_install_script_crashtracking = InstallerAutoInjectionScenario(
```

Comment:
I advocate for using the existing scenario "INSTALLER_AUTO_INJECTION" . And after break the app, restore the app

Reply:
Done